include/segment_symbols.h

```diff
@@ -648,7 +648,9 @@ DECLARE_ROM_SEGMENT(spot20_room_0)
 
 DECLARE_ROM_SEGMENT(ganon_tou_room_0)
 
-#if OOT_DEBUG
+// Room symbols for compiling test scenes, these are not wrapped in an
+// `#if OOT_DEBUG` so that the Debug ROM can be built with OOT_DEBUG=0.
```

Comment:
```suggestion
// `#if OOT_DEBUG` so that gc-eu-mq-dbg can be built with OOT_DEBUG=0.
```

Reply:
Should we start saying "building with debug features" instead of "the debug rom" or "gc-eu-mq-dbg"?
I would like to see support added for the other debug roms that we have soon, which I guess will be "gc-eu-dbg-1" and "gc-eu-dbg-2", or however we end up calling them. So we'd need a more general way to reference them in that case

Reply:
Changed to "debug ROMs (including gc-eu-mq-dbg)" to hopefully satisfy everyone here

Reply:
ngl I forgot there were other debug roms